app/default_overrides.go

```diff
@@ -117,7 +139,7 @@ type govModule struct {
 // DefaultGenesis returns custom x/gov module genesis state.
 func (govModule) DefaultGenesis(cdc codec.JSONCodec) json.RawMessage {
 	genState := govtypes.DefaultGenesisState()
-	genState.DepositParams.MinDeposit = sdk.NewCoins(sdk.NewCoin(BondDenom, sdk.NewInt(10000000)))
+	genState.DepositParams.MinDeposit = sdk.NewCoins(sdk.NewCoin(BondDenom, sdk.NewInt(100000000)))
```

Comment:
https://github.com/celestiaorg/celestia-app/pull/2416/files#r1315175128

IMO, 1000TIA should be the MinDeposit.

Reply:
https://github.com/celestiaorg/celestia-app/pull/2417/commits/996f410639444b8df20744bfe40cb26e7b00fbfa